TextImageParser.py

```diff
@@ -0,0 +1,43 @@
+import cv2
+import json
+import numpy as np
+import os
+
+'''
+Basic Process:
+If mode is line, then draw lines connecting the points and then draw a circle with center at each of those
+and with radius equal to the width.
+
+If mode is polygon, then use draw polygon fill the polygon.
+
+TODO: Add os iteration for text in folders
+'''
+
+if __name__ == "__main__":
+    file_path = # Put the path to the image file
+    read_file = open(file_path,"r")
+
+    data = json.load(read_file)
+    labels = data["labels"]
+    size = data["image_size"]
+
+    # print(len(labels))
+    img_mask = np.zeros((size[0],size[1]))
+    for lbl in labels:
+        if (lbl["mode"] == "POLYGON"):
+            pts = np.array(lbl["geometry"],np.int32)
+            pts = pts.reshape((-1,1,2))
+
+            img_mask = cv2.polylines(img_mask,[pts],True,255,3)
```

Comment:
Maybe use key-word arguments here to make it more obvious what the arguments mean